examples/process-example/process-example-camunda-7/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+server.port=5050
+camunda.bpm.client.base-url=http://localhost:8080/engine-rest
+camunda.autoDeploy.enabled: false
```

Comment:
I think that variable has no influence on anything since it's wrong formated => just delete it

Reply:
It actually does exactly what it says. You can checkout the example of Camunda community: https://github.com/camunda-community-hub/camunda-platform-7-rest-client-java.
Looks weird I know, but works.

Reply:
More concrete, this file: https://github.com/berndruecker/camunda-7-remote-spring-boot-example/blob/main/src/main/resources/application.properties

Reply:
Ok thanks. Could you add that as a comment, please?